src/Webhook/WebhookVerifier.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Fingerprint\ServerAPI\Webhook;
+
+final class WebhookVerifier
+{
+    public static function checkHeader(string $header, string $data, string $secret): bool
```

Comment:
Let's rename this method to something like `IsValidWebhookSignature` to keep it consistent with other SDKs.

Reply:
Hey 👋 Since you approved it, I mark this as resolved.